### DISCUSSION TITLE
Stop $ being url encoded

### DISCUSSION THREAD

**Discussion by SamPetherbridge** [encode/httpx#2724] [Q&A]
I am currently working with a system that is expecting a $ to be in a url parameter. I have tried a number of approaches however have not had any luck as yet. Is there any way to stop httpx from automatically encoding dollar signs as %24?

**Comment by lovelydinosaur** [comment]
It won't be escaped if you include it directly in the URL...

```python
>>> r = httpx.get("https://github.com/encode/httpx/discussions/2724?q=$")
>>> r.request.url
URL('https://github.com/encode/httpx/discussions/2724?q=$')
```

What's the use-case / requirement?
